build/shade/_k-standard-goals.shade

```diff
@@ -154,13 +154,15 @@ default SAMPLES_PROJECT_GLOB = "samples/*/*.csproj"
 
       projectGlobs.AddRange(srcProjects);
     }
+
     if (!BuildSrcOnly && Directory.Exists("test"))
     {
-        projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
+      projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
     }
-    if (!BuildSrcOnly && Directory.Exists("samples") && Directory.EnumerateFiles("samples", "*.csproj", SearchOption.AllDirectories).Any())
+
+    if (!BuildSrcOnly && Directory.Exists("samples"))
     {
-        projectGlobs.AddRange(Files.Include(SAMPLES_PROJECT_GLOB).ToList());
+      projectGlobs.AddRange(Files.Include(SAMPLES_PROJECT_GLOB).ToList());
```

Comment:
Note this line will still throw if `SAMPLES_PROJECT_GLOB` contains an entry that starts with a non-existent folder. One would hope those overriding this variable know what they're doing&hellip;